ui/components/multichain/network-manager/network-tabs.tsx

```diff
@@ -17,11 +18,10 @@ export const NetworkTabs = ({ initialTab }: { initialTab: string }) => {
   return (
     <Box>
       <ModalHeader
-        onBack={handleClose}
         onClose={handleClose}
         closeButtonProps={{ 'data-testid': 'modal-header-close-button' }}
       >
-        Networks
+        {t('bridgeSelectNetwork') ?? 'Select network'}
```

Comment:
Copy change to match figma.
<img width="477" height="309" alt="Screenshot 2025-08-22 at 21 02 36" src="https://github.com/user-attachments/assets/1ab585dc-2c97-4ac6-8164-f4d3a9955043" />